### PR TITLE
Rust 1.26

### PR DESCRIPTION
**Pull request by ptescher**
Add support for rust 1.26

### CODE REVIEW

File: 1.26/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
+FROM openshift/base-centos7
+
+# This image provides a Rust environment you can use to run your Rust
+# applications.
+
+MAINTAINER Michail Kargakis <kargakis@protonmail.ch>
```
Comment (kargakis) [inline 1.26/Dockerfile]: Change or remove this please

File: 1.26/Dockerfile.rhel7 [hunk 2]
```diff
@@ -0,0 +1,39 @@
+FROM rhscl/s2i-base-rhel7
+
+# This image provides a Rust 1.26 environment you can use to run your Rust
+# applications.
+
+MAINTAINER Michail Kargakis <kargakis@protonmail.ch>
```
Comment (kargakis) [inline 1.26/Dockerfile.rhel7]: Ditto

**Comment by kargakis** [comment]
@pat2man thanks for the PR, please make 1.26 the default in the Makefile.

**Comment by pat2man** [comment]
@kargakis Fixed.

**Comment by nak3** [comment]
Would you like to change rust version to more generic?

For examle, I guess that it is possible by using `ARG RUST_VERSION` (and replace all version by `RUST_VERSION`) in Dockerfile and by passing `--build-arg RUST_VERSION=1.26.2(or any version provided by rust-toolset-7)` at build time.

Since Rust's release cycle is very fast, it is difficult to follow version update.

(I'm sorry I don't want to bother you, so if I should open a new RFE for it outside of this PR#8, I will open it.)

**Comment by 0xmichalis** [comment]
I am not crazy about overgeneralizing on the slight posibility that there may be other diffs between versions in Dockerfiles in future versions that may not necessarily apply to older or newer.

**Pull request merged**